.gitignore

```diff
@@ -3,7 +3,5 @@ node_modules
 .ftpconfig
 .ftpignore
 .well-known
```

Comment:
What is this any why it's ignored?

Reply:
Some stuff that I have since deleted from the project + .well-know is used to verify the domain with Google.